ci-droid-gui/src/app/routes/form/form.component.spec.ts

```diff
@@ -433,11 +433,6 @@ describe('FormComponent', () => {
       expect(component.resources.length).toEqual(1);
     });
 
-    it('should create the request for bulk update', () => {
```

Comment:
if you are removing tis test then you shouldnt be doing 

`expect(ciDroidService.sendBulkUpdateAction).toHaveBeenCalledWith(component.createUpdateRequest());`

reason being you are not testing

`component.createUpdateRequest()`

Either you have  the unit test or assert it with the object

Example;

```JavaScript
const expectedRequest: BulkUpdateRequest = {
//todo
};
expect(ciDroidService.sendBulkUpdateAction).toHaveBeenCalledWith(expectedRequest);
```